README.md

```diff
@@ -31,20 +31,30 @@ The Javadocs are in the `./javadoc` directory.
 
 ## Relationship to the Iceberg Repository
 
-The `documentation` folder in the [Iceberg repository](https://github.com/apache/iceberg) contains all the markdown docs used by the website.
-The `common` subfolder contains contents used by the landing page.
-The `versioned` subfolder contains the contents different for each version.
+All markdown pages that are part of a versioned released are maintained in the iceberg repository. All pages common across all version releases are
+kept here in the iceberg-docs repo. A few exceptions are the markdown files that can be found in the `format` folder in the iceberg repository and contains
+markdown files that are copied to `./landing-page/content/common/format`.
```

Comment:
Unnecessary newline? Or maybe this is just wrapping at a different column than I was expecting.

Reply:
It looks like GitHub cuts a newline earlier than my IDE but it's not an actual newline in the rendered page. I broke this apart though for the sake of future PR reviews. 👍